src/cr/cube/cube.py

```diff
@@ -798,9 +810,6 @@ def _flat_values(self):
         if valid_counts:
             return np.array(valid_counts, dtype=np.float64)
 
-        if result["counts"] == [None]:
```

Comment:
Also not sure about this (I had previously introduced this line to avoid having to change the behavior of `cube_part::Nub::is_empty` - see change below).

This felt cleaner to me because it makes the return type more predictable.

Reply:
Hmm, a list containing the single value `None` seems odd. Do we have JSON that looks like `"counts": [null]` or something? That would be ugly indeed, as we could choose either `null` or an empty array and surely one of those would suit better.

Anyway, I'll see what I think after looking at the Nub bit, otherwise I'd say good riddance to this line :)